IA/IA_consume_files.py

```diff
@@ -55,16 +51,7 @@ def consume_files(guid, token, directory, retry=True):
 
     try:
         response = requests.get(zip_url.format(guid), headers=auth_header)
-        if response.status_code == 429:
-            keep_trying = retry
```

Comment:
Why did you remove all the retry logic?

Reply:
I removed it because it is not yet supported, but it would be a good idea to have it future-ready.

Reply:
could you use the the `get_with_retry` function in `utils.py`?

Reply:
I had to modify it to utilize headers, but as you wish.